src/config.c

```diff
@@ -100,6 +100,37 @@ struct Config *getCommonConfig(void) {
     return &commonConfig;
 }
 
+// Reloads the configuration file and removes interfaces which were removed from config.
+void reloadConfig() {
```

Comment:
Anyway, this function is not used in current pull request, so it it is not needed in this pull request and may be separated into other next pull requests (to have smaller changes).